### DISCUSSION TITLE
Docs regarding Prometheus Metric References

### DISCUSSION THREAD

**Discussion by onurccn** [woodpecker-ci/woodpecker#2916] [Ask a Question] [answered]
In docs regarding [Prometheus](https://woodpecker-ci.org/docs/administration/prometheus#metric-reference), the metric reference section has changed metric names in comments, but the old metric names on the actual metric example. Is this intentional? I can see the change [here](https://github.com/woodpecker-ci/woodpecker/pull/1224/files#diff-473b094525f84fa6479f28a05e95ed557ec0b737799f0f79b2258c2074ef51fbR38) but cant make sure if its intentional or just missed.

```
# HELP woodpecker_pipeline_count Pipeline count.
# TYPE woodpecker_pipeline_count counter
woodpecker_build_count{branch="main",pipeline="total",repo="woodpecker-ci/woodpecker",status="success"} 3
woodpecker_build_count{branch="mkdocs",pipeline="total",repo="woodpecker-ci/woodpecker",status="success"} 3
```

**Comment by qwerty287** [comment, marked as answer]
This rather looks like I missed to update these fields as well at #1224

**Reply by qwerty287** [reply]
See https://github.com/woodpecker-ci/woodpecker/pull/2919

**Reply by onurccn** [reply]
Thanks for the quick PR.
